ParisBNF/abb/BNFabb88.xml

```diff
@@ -0,0 +1,381 @@
+                                ቅድስናከ፡ ውስቴታ፤ ወይጸልለኒ፡ እምሰምይ፡ አክናፊከ፡ መንታ፨ </incipit>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf" quantity="182">182</measure>
+                                    <dimensions type="outer"/>
+                                    <height unit="mm" cert="medium">250</height>
+                                    <width unit="mm" cert="medium">200</width>
+                                </extent>
+                                <foliation>Foliation in the upper right corner of the
+                                    folia.</foliation>
+                                <collation>
+                                    <note>Due to the nature of modern library rebinding, the quire
+                                        quire structure has become invisible or lost.</note>
+                                </collation>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="18">
+                                    <ab type="pricking">Pricking is partly visible.</ab>
+                                    <ab type="ruling">Ruling is not visible.</ab>
+                                    <ab type="ruling" subtype="pattern" cert="low"
+                                        >1A-1A-1A1A/0-0/0-0/C</ab>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <seg type="script">18th-century script</seg>
+                                <seg type="ink">Black, red.</seg>
+                                <desc>Careful, irregular; by a good scribe. <persName
+                                        ref="PRS1071dAbbadi"/> describes the handwriting as <foreign
+                                        xml:lang="fr">ʻguịlḥ moderneʼ</foreign>. </desc>
+                            </handNote>
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="miniature">
+                                <locus target="#2v"/>
+                            </decoNote>
+                            <decoNote xml:id="d2" type="miniature">
+                                <locus target="#3v"/>
+                                <desc><ref type="authFile" corresp="AT1042HagiographicScene"
+                                            ><persName ref="PRS9848Walatta"/>Walatta Ṗeṭros taming
```

Comment:
the encoding `<persName ref="PRS9848Walatta"/>Walatta Ṗeṭros `does not make much sense, especially inside of a link created by `ref`